truckersmp-cli

```diff
@@ -134,11 +138,30 @@ else
 fi
 
 # extract md5sums and filenames
-sed -r  -e 's@\{"Files":\[\{@@g' \
-        -e  's@"FilePath":"([a-zA-Z0-9\/_\.-]+)","Type":"[a-zA-Z0-9]+",'\
-'"Md5":"([0-9a-f]{32})"[]\}\{,]{3}@\2 '"$moddir"'\1\n@g' \
-        < "$filelist" \
-        > "$checksums"
+# try downloaded "get_checksums" script if it exists,
+# or else the script in ${scriptdir}.
```

Comment:
I would do it the other way around, try the local (probably outdated) script first and then try to update it.